langchain4j-milvus/src/test/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStoreIT.java

```diff
@@ -57,6 +62,8 @@ void should_not_retrieve_embeddings_when_searching() {
                 .host(milvus.getHost())
                 .port(milvus.getMappedPort(19530))
                 .collectionName(COLLECTION_NAME)
+                .username("")
```

Comment:
are empty strings for user and pws intended?

Reply:
Yes, after updating the sdk, the username and password cannot be null. This test case can run normally when the Milvus username and password are not set.

Reply:
So when running locally, empty username and password are now mandatory? If so, can we set default values in ctor for username/pwd? with `getOrDefault(x, "")`

Reply:
To avoid breaking changes for existing users

Reply:
I forgot to set the default username and password in the constructor, add it now